xAPI.md

```diff
@@ -3551,145 +3550,16 @@ decision to use this scheme.
 
 See [Appendix C: Cross Domain Request Example](#Appendix3C) for an example. 
 
-<a name="datatransfer"/> 
-
-## 2.0 Data Storage and Retrieval
-
-This section contains implementation details and requirements surrounding how an LRS receives and responds to requests for data.  As mentioned in the previous section, this communication is done through HTTP 
-Requests to specific Resources, all of which have Endpoints. 
-
```

Comment:
The section below has moved, the few lines above are deleted. Why deleted? Because the existing section 2.2 intro is a much better intro to the whole section.